cast/src/main/java/com/ibm/wala/cast/ipa/cha/CrossLanguageClassHierarchy.java

```diff
@@ -276,5 +276,13 @@ public static CrossLanguageClassHierarchy make(AnalysisScope scope, ClassLoaderF
   public Set<TypeReference> getUnresolvedClasses() {
     return HashSetFactory.make();
   }
+
   /* END Custom change: unresolved classes */
+
+  @Override
+  public void clearCaches() {
+    for (IClassHierarchy cha : hierarchies.values()) {
+      cha.clearCaches();
+    }
```

Comment:
I'd prefer a tacit style here:

```suggestion
    hierarchies.values().forEach(IClassHierarchy::clearCaches);
```

OK to ignore this suggestion if you prefer the explicit loop.